app/batid/exceptions.py

```diff
@@ -8,3 +8,7 @@ class BANAPIDown(Exception):
 
 class BANBadResultType(Exception):
     """BAN result has not the expected type (ie 'voie' instead of 'numero')"""
+
+
+class PlotUnknown(Exception):
+    """The given plot id is not in our database"""
```

Comment:
Dans l'idée de faire du RNB un commun, on veut éviter de parler de "notre" base je pense.

```suggestion
    """The given plot id is not in RNB database"""
```

Reply:
ok